example/src/components/Login.vue

```diff
@@ -57,10 +57,13 @@ const handleLoginClick = () => {
   })
 }
 
-// these things are just here for the demonstration of updateName function
+// Demonstrate the updateName function
 const customName = ref(name.value)
 
-watch(name, (value) => (customName.value = value))
+watch(
+  () => name.value,
```

Comment:
I found that I had to use the callback signature of `watch` to make the compiler happy. I'm not sure exactly why, it's kind of unfortunate.